yarn.lock

```diff
@@ -16527,6 +16535,26 @@ __metadata:
   languageName: node
   linkType: hard
 
+"one-double-zero@npm:1.0.0-beta.14":
+  version: 1.0.0-beta.14
+  resolution: "one-double-zero@npm:1.0.0-beta.14"
+  dependencies:
+    "@bcoe/v8-coverage": ^0.2.3
+    "@jridgewell/trace-mapping": ^0.3.25
+    commander: ^12.1.0
+    find-up: ^5.0.0
+    glob: ^10.4.2
+    istanbul-lib-coverage: ^3.2.2
+    istanbul-lib-report: ^3.0.1
+    istanbul-reports: ^3.1.7
+    toml: ^3.0.0
+    typescript: ^5.5.2
```

Comment:
I don't think we should add a dependency that brings in typescript 🤔 I guess we could make the dep pluggable (i.e. optional peer dependency).

That would also handle the fact that it is minimum node v18 and Jest v30 supports Node 16